tests/unit/storage/test_prisma_adapter.py

```
@@ -200,6 +200,45 @@ async def test_migration_runner_applies_sequential_files(
+    mock_prisma, mock_tx_context, tmp_path, monkeypatch
+):
+    """' ; ' のような空ステートメントが execute_raw に渡されないことを検証。"""
+    from context_store.storage.prisma import _PrismaMigrationRunner
+
+    prisma_file = tmp_path / "src" / "context_store" / "storage" / "prisma.py"
+    prisma_file.parent.mkdir(parents=True, exist_ok=True)
+    monkeypatch.setattr("context_store.storage.prisma.__file__", str(prisma_file))
+
+    migrations_dir = tmp_path / "docker" / "postgres"
+    migrations_dir.mkdir(parents=True)
+    # セミコロンのみの行や、空白＋セミコロンの行を含む SQL
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `＋` (FULLWIDTH PLUS SIGN). Did you mean `+` (PLUS SIGN)?

(RUF003)